plugins/auto-sizes/optimization-detective.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Optimization Detective extensions by Auto Sizes.
+ *
+ * @since n.e.x.t
+ * @package auto-sizes
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Visits responsive lazy-loaded IMG tags to ensure they include sizes=auto.
+ *
+ * @since n.e.x.t
+ *
+ * @param OD_Tag_Visitor_Context $context Tag visitor context.
+ * @return bool Whether visited.
```

Comment:
```suggestion
 * @return false Whether the tag should be recorded in URL metrics.
```

See https://github.com/WordPress/performance/issues/1342.